index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Software Developer - Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="navcontainer">
+        <nav class="navbar">
+          <div class="logo">My Logo</div>
+          <div class="hamburger">
+            <i class="fa fa-bars"></i>
+          </div>
+
+          <ul class="menu menu-list">
+            <li>Portfolio</li>
+            <li>About</li>
+            <li>Contact</li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <section class="section">
+      <div class="sectionheading">
+        <p>I'm Collins.</p>
+        <p>Glad to see you!</p>
```

Comment:
- [ ] Good job so far :clap: , however, as part of the project requirement, this section supposes to be wrapped in a `h1` tag for `html` semantic purposes please fix it. :slightly_smiling_face: